src/expr/src/interpret.rs

```diff
@@ -641,9 +641,9 @@ impl Interpreter for Trace {
             return Pushdownable::No;
         }
 
-        let is_monotone = func.is_monotone();
+        let pushdownable_fn: Pushdownable = func.is_monotone().into();
```

Comment:
The `BinaryFunc` case can differentiate the `pushdownable` value for the first and second parameter, but `variadic` returns a single value. I assume this means that we will only return `true` for `VariadicFunc` symbols that are `pushdownable` in _all_ parameters.

Reply:
Right, yeah. Or, at least, monotone in all parameters... and the only variadic functions we know how to push down are the monotone ones.